modules/asset-manager-impl/pom.xml

```diff
@@ -66,6 +66,11 @@
       <artifactId>opencast-authorization-manager</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.opencastproject</groupId>
+      <artifactId>opencast-scheduler-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
We should think about possible side effects here.
Thinking on potential for circular dependencies

Reply:
I could easily remove this dependency. It was only used to get the string value of the the recording state 'unknown', which isn't expected to change.